docs/getting_started_guide/Installation_Options/Repository/Installing_SIMP_From_A_Repository.rst

```diff
@@ -120,5 +75,5 @@ can discover what extra modules are available by searching for ``pupmod`` via
 .. _EPEL Repositories: https://fedoraproject.org/wiki/EPEL
 .. _Google Cloud: https://cloud.google.com
 .. _Microsoft Azure: https://azure.microsoft.com
-.. _official SIMP YUM repositories: https://packagecloud.io/simp-project
+.. _official SIMP YUM repositories: https://download.simp-project.com/simp/yum
```

Comment:
This is much better nomenclature than "SIMP Download Server" when discussing the RPMs